docs/guides/getting-started-java-spring.md

```diff
@@ -36,7 +36,7 @@ While stepping through this guide, you can visit our [sample repository](https:/
 Before getting started, ensure you have the following in your local environment:
 
 - Access to your preferred code editor or IDE
-- [OpenJDK 21+](https://openjdk.org/install/)
+- [OpenJDK 21+](https://openjdk.org/install/) and [Maven 3](https://maven.apache.org/index.html)
```

Comment:
I installed the JDK differently this time (https://mise.jdx.dev/lang/java.html) than previous times running through this guide. I'm not sure if the requirement to install Maven has always been separate, or if it depends on how you install a JDK.

Reply:
Good add. I would assume a Java dev would have Maven, but it's not packaged with the JDK.